ci/pipelines/default-pipeline.yml

```diff
@@ -39,15 +44,27 @@ stages:
     - if [[ "$exit_code" -ne 0 ]]; then exit 1; fi
     - exit 0
   set-publishing-credentials:
-    - aws ssm get-parameter --region us-east-1 --name ci.dd-sdk-android.gradle-properties --with-decryption --query "Parameter.Value" --out text >> ./gradle.properties
-    - export GPG_PRIVATE_KEY=$(aws ssm get-parameter --region us-east-1 --name ci.dd-sdk-android.signing.gpg_private_key --with-decryption --query "Parameter.Value" --out text)
-    - export GPG_PASSWORD=$(aws ssm get-parameter --region us-east-1 --name ci.dd-sdk-android.signing.gpg_passphrase --with-decryption --query "Parameter.Value" --out text)
-    - export CENTRAL_PUBLISHER_USERNAME=$(aws ssm get-parameter --region us-east-1 --name ci.dd-sdk-android.publishing.central_username --with-decryption --query "Parameter.Value" --out text)
-    - export CENTRAL_PUBLISHER_PASSWORD=$(aws ssm get-parameter --region us-east-1 --name ci.dd-sdk-android.publishing.central_password --with-decryption --query "Parameter.Value" --out text)
-    - export GPG_PUBLIC_FINGERPRINT=$(aws ssm get-parameter --region us-east-1 --name ci.dd-sdk-android.signing.gpg_public_key --with-decryption --query "Parameter.Value" --out text | gpg --import --import-options show-only | grep -E -o -e "[A-F0-9]{40}")
+    - cp ./ci/pipelines/secrets/gradle.properties ./gradle.properties
+    - export GPG_PRIVATE_KEY=$(cat ./ci/pipelines/secrets/gpg_private_key)
+    - export GPG_PASSWORD=$(cat ./ci/pipelines/secrets/gpg_passphrase)
+    - export CENTRAL_PUBLISHER_USERNAME=$(cat ./ci/pipelines/secrets/central_username)
+    - export CENTRAL_PUBLISHER_PASSWORD=$(cat ./ci/pipelines/secrets/central_password)
+    - export GPG_PUBLIC_FINGERPRINT=$(cat ./ci/pipelines/secrets/gpg_public_key | gpg --import --import-options show-only | grep -E -o -e "[A-F0-9]{40}")
 
 # CI IMAGE
 
+fetch-secrets:
+  stage: fetch-secrets
+  tags: ["macos:sonoma","specific:true"]
+  image: $CI_IMAGE_DOCKER
+  script:
+    - !reference [.snippets, fetch-secrets]
+  artifacts:
+    paths:
+      - ./ci/pipelines/secrets/
+    expire_in: 1 hour
+    when: always
```

Comment:
Is it a safe thing to have? it means that anyone with just a basic Gitlab CI access can download this, not only RUM team members.

Moreover, it makes this artifact available for 1 hour, meaning it is still there even after pipeline completed, failed.

Should we maybe fetch secrets only in the boundaries of the particular job?